### ISSUE TITLE
Alert when database backup job failed

### ISSUE THREAD

**Issue by billmetangmo** [mongulu-cm/manzi-mfa#64]
https://record.reverb.chat/s/wZqdo21L15erxeiD2NOG

Solutions:
- https://github.com/billmetangmo/descartes-devops/blob/088efb9db447f3e0e36d3acf4352f3474fae5c53/Taskfile.yml#L28 + https://github.com/mongulu-cm/manzi-mfa/blob/main/manifests/database-backup.yml (rajouter un token github pour ouvrir l'issue)
- https://ramitsurana.github.io/awesome-kubernetes/projects/projects/#monitoring-services
- https://healthchecks.io/

**Comment by fabiolatagne97** [comment]
@billmetangmo l' issue ne contient aucun label. L'email automatique de relai n'a donc pas été envoyé. D'ici 3 jours si aucun label n'est ajouté , il n'y aura aucune communication automatique sur l'issue

**Comment by fabiolatagne97** [comment]
@billmetangmo merci d'avoir remli les tags à temps.
Un email automatique a été envoyé :) .
N'oublie pas d'envoyer également un message dans le groupe whatsapp.

**Comment by billmetangmo** [comment]
https://github.com/mongulu-cm/manzi-mfa/blob/main/manifests/database-backup.yml

**Comment by billmetangmo** [comment]
https://github.com/billmetangmo/descartes-devops/blob/088efb9db447f3e0e36d3acf4352f3474fae5c53/Taskfile.yml#L28

**Comment by billmetangmo** [comment]
Healtcheck has no direct integration to github and i don't want to add additional services so the first solution is preferred

**Comment by billmetangmo** [comment]
La première solution est meilleure mais le job au sens k8s sera success alors que failed dans la réalité donc KO

**Comment by billmetangmo** [comment]
It works but I update external-ecrets user with a policy that allows to wrte to te database. I will need to chnage it to use a role instead inspired by https://medium.com/@th3b3ginn3r/allowing-kubernetes-service-account-to-assume-iam-role-dbfec552a6a6

**Comment by tsafacjo** [comment]
is this issue still open ?

**Comment by billmetangmo** [comment]
Yes it's still open, I need to change https://github.com/mongulu-cm/manzi-mfa/issues/64#issuecomment-2526216241
